src/components/passphraseBackup/index.js

```diff
@@ -0,0 +1,136 @@
+    const data = {
+      ...this.props,
+      qrcode: this.canvasRef.firstChild.toDataURL(),
+    };
+    renderPaperwallet(data, this.walletName);
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this.timeout);
+  }
+
+  setCanvasRef(node) {
+    this.canvasRef = node;
+  }
+
+  render() {
+    const {
+      t, account,
+    } = this.props;
+    const { passphraseCopied } = this.state;
+
+    return (
+      <React.Fragment>
+        <div className={`${styles.optionsHolder}`}>
+          <div className={`${styles.option}`}>
+            <div className={`${styles.optionContent}`}>
+              <h2>
+                {t('Passphrase')}
+
+                <Tooltip
+                  title={'Save the passphrase'}
+                  footer={
+                    <a href={links.howToStorePassphrase}
+                      rel="noopener noreferrer"
+                      target="_blank">
+                        {t('Read More')}
+                    </a>}>
+                  <p>{
+                    t('Store your passphrase in a safe place, possibly having more than one copy. You can use a password manager or a paperwallet. It is very important to ensure you do not lose access to your passphrase.')
+                  }</p>
+                </Tooltip>
+
+              </h2>
+              <div className={`${styles.inputs} ${grid.row} passphrase`}>
+                {account.passphrase.split(' ').map((value, i) => (
+                  <span key={i} className={`${grid['col-xs-2']}`}>
+                    <InputV2
+                      readOnly
+                      value={value}
+                    />
+                  </span>
+                ))}
+              </div>
+              <CopyToClipboard
+                text={account.passphrase}
+                onCopy={() => this.textIsCopied()}>
+                <span className={`${styles.action} ${passphraseCopied && styles.copied}`}>
+                  { !passphraseCopied ? t('Copy to Clipboard') : t('Copied!') }
+                </span>
+              </CopyToClipboard>
+            </div>
+          </div>
+          <div className={styles.hrSection}>
+            <p>{t('OR')}</p>
+          </div>
+          <div className={`${styles.option}`}>
+            <div className={`${styles.optionContent}`}>
+              <h2>
+                {t('Paper Wallet')}
+                <Tooltip
+                  title={'Paper Wallet'}>
+                  <p>
+                    {t('You can print your passphrase to store in a safe place. ')}
+                    {t('It is highly recommended to delete the PDF file and remove it from your Trash Folder too after printing it.')}
+                  </p>
+                </Tooltip>
+              </h2>
+              <div style={{ display: 'none' }} ref={this.setCanvasRef}>
+                <QRCode value={account.passphrase} />
+              </div>
+              <div className={styles.downloadLisk}>
+                <img src={svgIcons.fileOutline} />
```

Comment:
If I am not wrong I think we can use here the component that you use for manage the svgs